### DISCUSSION TITLE
[Question] Change background color on loading

### DISCUSSION THREAD

**Discussion by lrrrtm** [flet-dev/flet#3389] [Q&A]
### Question

Hi! I want to find out if it is possible to change the background color during the download of the application (gray color in the screenshot)
![image](https://github.com/flet-dev/flet/assets/74620760/db8f0613-51b2-4aba-ba50-0e2318b166cb)

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by joaocassianox7x** [comment]
Someone figure out how to solve it ? I'm using the 0.24 version and keep with same issue. Nothing about how to change in documentation.
